packages/cache/src/internal/cacheUtils.ts

```diff
@@ -116,9 +120,12 @@ export function getCacheFileName(compressionMethod: CompressionMethod): string {
     : CacheFilename.Zstd
 }
 
-export async function isGnuTarInstalled(): Promise<boolean> {
+export async function getGnuTarPathOnWindows(): Promise<string> {
```

Comment:
Looks like this function may be called multiple times. We can cache the result in a variable in this file.

Reply:
It is called only two times

Reply:
Ok